examples/basic_tracer/tracer.py

```diff
@@ -26,12 +26,23 @@
 if os.getenv("EXPORTER") == "jaeger":
     from opentelemetry.ext.jaeger import JaegerSpanExporter
 
+    print("Using JaegerSpanExporter")
     exporter = JaegerSpanExporter(
         service_name="basic-service",
         agent_host_name="localhost",
         agent_port=6831,
     )
+elif os.getenv("EXPORTER") == "collector":
```

Comment:
Not a problem for this PR, but I think it'd be better to have multiple files in the example than set the exporter based on an env var.

Reply:
Added logging exporter in config

Reply:
@c24t actually I think it is better to keep everything in a single file and configure different exporters based on env variables / arg parameters. It'll avoid us the maintenance cost over different files and would keep them aligned. Do you have any specific reason to prefer different files?

Reply:
Just to avoid having env vars affect the example behavior, but I agree it's a tradeoff. I was thinking one main module in the example and separate modules for each type of exporter that call into main.